### ISSUE TITLE
Switching between week and month view is confusing

### ISSUE THREAD

**Issue by thatoddmailbox** [MyHomeworkSpace/client#174]
On the calendar, switching between week and month view is a bit weird.

For example: it's currently May 2021. If you go to month view, you will see the month of May. If you then switch to week view, it takes you to the first week of May, which is the week of April 26th. That means if you click the month view button, it'll take you to the month of April. If you click week view again, it'll take you to the first week of April, which is the week of March 29th. If you then click month view, you'll get the month of March.

In other words: go to calendar, click month then week then month then week then month and wonder how you ended up in March 2021.

I'm not actually sure how Real Calendars™ handle this, but it seems like going from month to week should somehow bring you back to the week you were looking at before?

**Comment by willbarkoff** [comment]
> I'm not actually sure how Real Calendars™ handle this, but it seems like going from month to week should somehow bring you back to the week you were looking at before?

That's exactly how Calendar.app and Google Calendar seem to handle it
